### ISSUE TITLE
15.0 VS 18.3

### ISSUE THREAD

**Issue by Nikolaev-Nikolay** [ethereum-mining/ethminer#1822]
![data_eth](https://user-images.githubusercontent.com/40342151/51099451-9c1a3b00-17f2-11e9-9c33-d75932f7985d.png)
I tried to go from 18.2 to 15.0 and here is the result. Jerks less, volume more. Guys the code is clearly not there. What do you think is going on?! Frequency and load one.

**Comment by chfast** [comment]
We need more information: ethminer versions, OS, GPUs, command line options.
I have not idea what those graph show.

**Comment by chfast** [comment]
But what's the difference between 0.18 and 0.15?

**Comment by Nikolaev-Nikolay** [comment]
Hashrate difference:
0.18 ~ 76 (MHs)
0.15 ~  116.83...120 (MHs)
CUDA 10
7...13 Jan 0.18
13..14 Jan 0.15
Curve keep anything down, not constant load. 0.15 behaves smoothly and loaded cards well gut.

**Comment by chfast** [comment]
Did you check other versions?

**Comment by Nikolaev-Nikolay** [comment]
No, I did not check, or rather as the new version I reinstalled ethminer, but it only worsened the situation. In the end I went back to 0.15.
 Want to know what was changed so indigenous in these versions.
Also

0.15->0.16->0.17->0.18 ?!
The Best hashrate?!
Or no?!

**Comment by AndreaLanfranchi** [comment]
Try 0.18 with --ergodicity CLI argument set to 2
Like this
```
ethminer.exe -U -P stratum1+tcp://0x1234567890000000000000@eth-ru.dwarfpool.com:8008/nngeo --ergodicity 2
```
please report when able.

**Comment by AndreaLanfranchi** [comment]
> Hashrate difference:
0.18 ~ 76 (MHs)
0.15 ~ 116.83...120 (MHs)

Are you talking about **computed** hashrate or "reported" hashrate ?

**Comment by Nikolaev-Nikolay** [comment]
dwarfpool.com report  every hoarse ago.
![2](https://user-images.githubusercontent.com/40342151/51127541-82571300-1847-11e9-831b-17dcb0705aee.png)

**Comment by AndreaLanfranchi** [comment]
Sorry ... there is some confusion here.
I can't understand which hashrate you're talking about.

**Comment by Nikolaev-Nikolay** [comment]
ok, it is   "reported" hashrate.

computed it's normal nvideo card summa 104...106 MH/s - it is normal in 0.15 and 0.18

**Comment by AndreaLanfranchi** [comment]
Ok then disregard my suggestion about `--ergodicity` CLI argument.

**Comment by Nikolaev-Nikolay** [comment]
Ok...
I am Sorry … what do it function "--ergodicity 2"?!

**Comment by AndreaLanfranchi** [comment]
> I am Sorry … what do it function "--ergodicity 2"?!

It picks a new random nonce search segment every time a job arrives.

**Comment by Nikolaev-Nikolay** [comment]
Here are the stats from the pool:

![eth](https://user-images.githubusercontent.com/40342151/51259510-25d23000-19ce-11e9-9a51-c93141384f7c.png)
--ergodicity CLI argument does not improve collection
the leader is 0.15
What else can try?! Let the guys like  on Linux or Windows? Can to move-where how to build a good advise.
How does a pool work from the inside?!
